pkgs/development/node-packages/node-packages-v12.nix

```diff
@@ -2047,13 +2047,13 @@ let
         sha512 = "w2VwSrBoHa5BsSyH+KxEqeQBAllHhccyMFVHtGtdMpF4W7IRWfZjFiQceJPChOeTsSDVUpER2T8FA93pr0L+QA==";
       };
     };
-    "temp-0.9.0" = {
+    "temp-0.9.1" = {
```

Comment:
I think this change is unrelated, you added a package to the v10 list, but an entry was updated in the v12 list